README-idp.md

```diff
@@ -0,0 +1,192 @@
+Idp module
+============
+
+Description
+-----------
+
+The idp module allows to ensure presence and absence of idps.
+
```

Comment:
I'm not sure if I understood the README correctly, but it seems that each provider has some specific required configuration. This make a few examples in the README look the same, only varying the value of the parameter for the same action (e.g. ensure provider is present).

Maybe it would be nice to have some description on why this is the case for this module (something we didn't have before), and this explanation seem to be a good fit for a "Use Cases" section.